### DISCUSSION TITLE
After update can not locate adapter any more

### DISCUSSION THREAD

**Discussion by hocomadvies** [Koenkk/zigbee2mqtt#13256] [General]
After I updated to 1.26.0 I can not start zigbee2mqtt any more. The reason is that the serial port can not be openend. So I tried to solve that plugin in and out the stick. Which worked, but only for a brief period (few minutes). Than I decided to add a powered usb hub to see if that would help. 
No I am no longer able to locate the adapter, to use the right serial port settings in my config.yaml

`Error: Error while opening serialport 'Error: Error: No such file or directory, cannot open /dev/ttyUSB0'
`
`pi@raspberrypi:/opt/zigbee2mqtt $ ls -l /dev/ttyACM0
ls: cannot access '/dev/ttyACM0': No such file or directory
`
How to solve this? Is it a matter of user / rights? I am running on a Rpi3b+

`pi@raspberrypi:/opt/zigbee2mqtt $ lsusb
Bus 001 Device 006: ID 2109:2811 VIA Labs, Inc. Hub
Bus 001 Device 005: ID 090c:1000 Silicon Motion, Inc. - Taiwan (formerly Feiya Technology Corp.) Flash Drive
Bus 001 Device 004: ID 0424:7800 Standard Microsystems Corp. 
Bus 001 Device 003: ID 0424:2514 Standard Microsystems Corp. USB 2.0 Hub
Bus 001 Device 002: ID 0424:2514 Standard Microsystems Corp. USB 2.0 Hub
Bus 001 Device 001: ID 1d6b:0002 Linux Foundation 2.0 root hub
`

`Jul 23 13:15:43 raspberrypi systemd[1]: Started zigbee2mqtt.
Jul 23 13:15:46 raspberrypi npm[11245]: > zigbee2mqtt@1.26.0 start
Jul 23 13:15:46 raspberrypi npm[11245]: > node index.js
Jul 23 13:15:52 raspberrypi npm[11245]: Zigbee2MQTT:info  2022-07-23 13:15:52: Logging to console and directory: 'data/log/2022-07-23.13-15-52' filename: log.txt
Jul 23 13:15:52 raspberrypi npm[11245]: Zigbee2MQTT:info  2022-07-23 13:15:52: Starting Zigbee2MQTT version 1.26.0 (commit #bc4ffc00)
Jul 23 13:15:52 raspberrypi npm[11245]: Zigbee2MQTT:info  2022-07-23 13:15:52: Starting zigbee-herdsman (0.14.40)
Jul 23 13:15:53 raspberrypi npm[11245]: Zigbee2MQTT:error 2022-07-23 13:15:53: Error while starting zigbee-herdsman
Jul 23 13:15:53 raspberrypi npm[11245]: Zigbee2MQTT:error 2022-07-23 13:15:53: Failed to start zigbee
Jul 23 13:15:53 raspberrypi npm[11245]: Zigbee2MQTT:error 2022-07-23 13:15:53: Check https://www.zigbee2mqtt.io/guide/installation/20_zigbee2mqtt-fails-to-start.html for possible solutions
Jul 23 13:15:53 raspberrypi npm[11245]: Zigbee2MQTT:error 2022-07-23 13:15:53: Exiting...
Jul 23 13:15:54 raspberrypi npm[11245]: Zigbee2MQTT:error 2022-07-23 13:15:54: Error: Error while opening serialport 'Error: Error: No such file or directory, cannot open /dev/ttyUSB0'
Jul 23 13:15:54 raspberrypi npm[11245]:     at SerialPort.<anonymous> (/opt/zigbee2mqtt/node_modules/zigbee-herdsman/src/adapter/z-stack/znp/znp.ts:146:28)
Jul 23 13:15:54 raspberrypi npm[11245]:     at SerialPort._error (/opt/zigbee2mqtt/node_modules/@serialport/stream/lib/index.js:198:14)
Jul 23 13:15:54 raspberrypi npm[11245]:     at /opt/zigbee2mqtt/node_modules/@serialport/stream/lib/index.js:242:12
Jul 23 13:15:54 raspberrypi systemd[1]: zigbee2mqtt.service: Main process exited, code=exited, status=1/FAILURE
Jul 23 13:15:54 raspberrypi systemd[1]: zigbee2mqtt.service: Failed with result 'exit-code'.
Jul 23 13:15:54 raspberrypi systemd[1]: zigbee2mqtt.service: Service RestartSec=100ms expired, scheduling restart.
Jul 23 13:15:54 raspberrypi systemd[1]: zigbee2mqtt.service: Scheduled restart job, restart counter is at 252.
Jul 23 13:15:54 raspberrypi systemd[1]: Stopped zigbee2mqtt.
`

**Comment by lux73** [comment]
it seems your Setup cannot reach your Stick ->

`pi@raspberrypi:/opt/zigbee2mqtt $ ls -l /dev/ttyACM0 ls: cannot access '/dev/ttyACM0': No such file or directory`

i assume it's an old crappy CC2531?  Either your USB Port have changed or your Stick died...

if you use Docker it can be that there is a Problem to reach the SAtick from inside thje Container, but i'm not familar with Docker. So i could not help with these

pls consider to upgrade your Stick to CC2652xx if you using old CC2531 - the old one are no more good enough these Days

**Reply by hocomadvies** [reply]
Thanks for the reply. It is working again after I unplugged from the usb/powered hub and stick it in the rpi3 again. I have a CC2652xx recently flashed new stick. For now I leave it and see how it runs (and if it stays running over time).

**Reply by lux73** [reply]
that's nice to hear...!

what exact CC2652 do you have? i've never seen one which works with **/dev/ttyACMx** - and i thought all CC2652 have **/dev/ttyUSBx**

if you have a USB Extension Cable laying around pls extend your Stick with these - directly connected with the USB Port it could lead in massive delay and other weird behaviour
